exception FatalErrorException of ()

fun abort {a:vt@ype} (): a